### PR TITLE
Contributing: fix typos

### PR DESCRIPTION
**Pull request by MaherJendoubi**

### CODE REVIEW

File: onnxruntime/core/protobuf/onnx.in.proto [hunk 1]
```diff
@@ -133,10 +133,10 @@ message AttributeProto {
 
   // The type field MUST be present for this version of the IR.
   // For 0.0.1 versions of the IR, this field was not defined, and
-  // implementations needed to use has_field hueristics to determine
+  // implementations needed to use has_field heuristics to determine
```
Comment (snnn) [inline onnxruntime/core/protobuf/onnx.in.proto]: The files under onnxruntime/core/protobuf are copied from https://github.com/onnx/onnx. Please send your changes there.

Comment (MaherJendoubi) [reply onnxruntime/core/protobuf/onnx.in.proto]: @snnn Thank you for the review.
Done:  https://github.com/onnx/onnx/pull/2571

File: server/protobuf/onnx-ml.proto [hunk 2]
```diff
@@ -136,10 +136,10 @@ message AttributeProto {
 
   // The type field MUST be present for this version of the IR.
   // For 0.0.1 versions of the IR, this field was not defined, and
-  // implementations needed to use has_field hueristics to determine
+  // implementations needed to use has_field heuristics to determine
```
Comment (snnn) [inline server/protobuf/onnx-ml.proto]: Could you please exclude this file?

Comment (MaherJendoubi) [reply server/protobuf/onnx-ml.proto]: @snnn done

**Comment by snnn** [comment]
Thank you!

**Pull request merged**